packages/@react-spectrum/s2/src/TagGroup.tsx

```diff
@@ -111,7 +115,7 @@ function TagGroup<T extends object>(
     helpText =  (
       <Text
         slot="description"
-        className={helpTextStyles({size: props.size || 'M'})}>
+        styles={helpTextStyles({size: props.size || 'M'})}>
```

Comment:
```suggestion
        styles={helpTextStyles({size})}>
```
just a spot where we could easily make a mistake and diverge unintentionally